packages/bot-web-ui/src/pages/bot-builder/quick-strategy/__tests__/quick-strategy.spec.tsx

```diff
@@ -252,18 +252,6 @@ describe('<QuickStrategy />', () => {
         });
     });
 
-    it('It should submit the form', async () => {
-        render(<QuickStrategy />, {
-            wrapper,
-        });
-
-        await waitFor(async () => {
-            await userEvent.click(screen.getByTestId('qs-run-button'));
-        });
-
-        expect(mock_DBot_store?.quick_strategy?.is_open).toBeFalsy();
-    });
```

Comment:
removed this since we dint have a submit form handler on the the modal initial view